bundles/org.openhab.binding.enturno/src/main/java/org/openhab/binding/enturno/internal/EnturNoHandler.java

```diff
@@ -0,0 +1,306 @@
+/**
+ * Copyright (c) 2014,2019 by the respective copyright holders.
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.enturno.internal;
+
+import com.google.gson.JsonSyntaxException;
+import org.apache.commons.lang.StringUtils;
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+import org.eclipse.jetty.client.HttpClient;
+import org.eclipse.smarthome.core.library.types.DateTimeType;
+import org.eclipse.smarthome.core.library.types.StringType;
+import org.eclipse.smarthome.core.thing.*;
+import org.eclipse.smarthome.core.thing.binding.BaseThingHandler;
+import org.eclipse.smarthome.core.thing.type.ChannelKind;
+import org.eclipse.smarthome.core.types.Command;
+import org.eclipse.smarthome.core.types.RefreshType;
+import org.eclipse.smarthome.core.types.State;
+import org.eclipse.smarthome.core.types.UnDefType;
+import org.openhab.binding.enturno.internal.connection.EnturCommunicationException;
+import org.openhab.binding.enturno.internal.connection.EnturConfigurationException;
+import org.openhab.binding.enturno.internal.connection.EnturNoConnection;
+import org.openhab.binding.enturno.internal.model.simplified.DisplayData;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.ScheduledFuture;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * The {@link EnturNoHandler} is responsible for handling commands, which are
+ * sent to one of the channels.
+ *
+ * @author Michal Kloc - Initial contribution
+ */
+@NonNullByDefault
+public class EnturNoHandler extends BaseThingHandler {
+
+    private final Logger logger = LoggerFactory.getLogger(EnturNoHandler.class);
+
+    private final HttpClient httpClient;
+
+    private @NonNullByDefault({})
+    EnturNoConfiguration config;
```

Comment:
Usually the eclipse formatter would have collapsed this into one line. Have you auto-formatted with the ESH formatter profile? If not please do so.

Reply:
Done